deep-security-agent/tasks/activate-linux.yml

```diff
@@ -0,0 +1,81 @@
+# Copyright 2018, Trend Micro
+#
+# License as per [repo](master/LICENSE)
+#
+# *********************************************************************
+# * Activate the Deep Security Agent (For Linux only)
+# *********************************************************************
+---
+# *********************************************************************
+# * Check if DSA installed properly
+# *********************************************************************
+- debug:
+    msg: "Check if DSA installed properly."
+
+- stat:
+    path: /opt/ds_agent/dsa_query
+  register: dsaQuery
+
+- fail:
+    msg: 
+        - "Could not find command /opt/ds_agent/dsa_query to query DSA certificate."
+        - "Please make sure Deep Security Agent has been installed"
+  when: dsaQuery.stat.size is not defined 
+
+# *********************************************************************
+# * Check if DSA activated already
+# *********************************************************************
+- debug:
+    msg: "Check if DSA activated already."
+
+- set_fact:
+    isActivated: false
+
+- shell: /opt/ds_agent/dsa_query -c GetAgentStatus agentCertHash
+  register: certHash
+  become: true
+
+- set_fact:
+    isActivated: true
+  when: (certHash.stdout | regex_search('(D0:68:DE:48:1B:DC:4D:00:89:07:7F:85:15:20:C5:90:11:EB:56:6A)')) != "D0:68:DE:48:1B:DC:4D:00:89:07:7F:85:15:20:C5:90:11:EB:56:6A"
```

Comment:
use > to make it synchronized

Reply:
After discussion, remove ">" to avoid confusing.